assets/js/standings_chart.js

```diff
@@ -0,0 +1,28 @@
+import Chart from "chart.js"
```

Comment:
Missing semicolon.

[Read more about it here.](http://eslint.org/docs/rules/semi)